README.md

```diff
@@ -1 +1,58 @@
-# JavaScript-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript group capstone
+
+
+In this project I have used linter for HTML and CSS and Javascript, to make sure there are no linter errors on the code.
```

Comment:
- [ ] Kindly, add a more descriptive title and description for your project to make a professional `README.md` file 👌
> For Example 💡

> Movies App
> In this project, We Created a webapp that displays movies and their descriptions ........